src/app/models/usermodel.d.ts

```diff
@@ -6,12 +6,12 @@ export type UserModelType = {
     first_name: string;
     groups: string[];
     last_name: string;
+    instructorEligible: boolean;
     pending_verification: boolean;
     stale_verification: boolean;
-    needsProfileCompleted: boolean;
+    incompleteSignup: boolean;
     pendingInstructorAccess: boolean;
     emailUnverified: boolean;
-    rejectedFaculty: boolean;
     is_newflow: boolean; // Obsolete?
```

Comment:
Not necessary here but you can remove this, there is no reason to check `is_newflow` anymore.